tests/core/recipes/compiled_recipes/test_real_orca_recipes.py

```diff
@@ -1,12 +1,18 @@
 from shutil import which
+import os
 
 import pytest
 from ase.build import molecule
 from numpy.testing import assert_allclose
 
 from quacc.recipes.orca.core import relax_job, static_job
 
-has_orca = bool(which("orca"))
+orca_path = which("orca")
```

Comment:
Can you change this to `which(SETTINGS.ORCA_CMD)`? This was something I should have done in the first place to not hard-code the value. See [here](https://github.com/Quantum-Accelerators/quacc/blob/722e9dd1cc8b2bc0956aae6e291ca044c103ad71/src/quacc/settings.py#L120-L126) for reference.

Reply:
Thanks. I have a new commit.

Reply:
Thanks. Looks like you're missing the import statement (`from quacc import SETTINGS`).

Reply:
I also install `deepsource` to my repo. But `deepsource` permission is still failing.